src/Microsoft.DotNet.Wpf/src/PresentationFramework/Microsoft/Win32/OpenFileDialog.cs

```diff
@@ -230,14 +190,11 @@ public bool ShowReadOnly
         {
             get
             {
-                // OFN_HIDEREADONLY is the inverse of our property,
-                // so negate the results of GetOption...
-                return !GetOption(NativeMethods.OFN_HIDEREADONLY);
+                return _showReadOnly;
             }
             set
             {
-                // ... and SetOption.
-                SetOption(NativeMethods.OFN_HIDEREADONLY, !value);
+                _showReadOnly = false;
```

Comment:
Can you add a comment here as well explaining the reason for doing this.

Reply:
Reverted in https://github.com/dotnet/wpf/pull/7916/commits/38ab72d13467b15166947fb783a5cb7ae4bd0079 (but still with no effect)